### PR TITLE
Delete draft confirmation added 

### PR DESCRIPTION
**Pull request by jesseyuen**
Screenshot_2020-12-07 Delete this draft record - Register trainee teachers - GOV UK(1)

### CODE REVIEW

File: app/views/new-record/delete-draft/confirm.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{% extends "_templates/_record-form.html" %}
```
Comment (edwardhorsford) [inline app/views/new-record/delete-draft/confirm.html]: This is the template for existing records, not draft ones. When swapping over, check there's nothing else we're missing.

File: app/views/new-record/delete-draft/confirm.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+{% set pageHeading = "Delete this draft record" %}
+{% set backLink = './../overview' %}
+{% set backText = "Back to draft record" %}
+{% set gridColumn = 'govuk-grid-column-full' %}
```
Comment (edwardhorsford) [inline app/views/new-record/delete-draft/confirm.html]: We don't want to override the default grid column

File: app/views/new-record/delete-draft/confirm.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+{% set formAction = "./../overview" | orReferrer(referrer) %}
```
Comment (edwardhorsford) [inline app/views/new-record/delete-draft/confirm.html]: No need for an action

**Review by edwardhorsford — CHANGES_REQUESTED** [review]
Looking good.

The cancel link goes to the wrong place because of it using the wrong template.

File: app/views/_templates/_new-record.html [hunk 4]
```diff
@@ -67,6 +67,9 @@
         {% if returnLink %}
           <p class="govuk-body"><a href="{{returnLink.href}}" class="govuk-link">{{returnLink.text}}</a></p>
         {% endif %}
+        {% if cancelLink %}
+          <p class="govuk-body"><a href="{{recordPath}}" class="govuk-link">Cancel and return to record</a></p>
+        {% endif %}
```
Comment (edwardhorsford) [inline app/views/_templates/_new-record.html]: I don't think this is needed. The above `returnLink` handles it already.

In the view itself you can do:
`{% set returnLink = { href: 'foo', text: 'Cancel and return to record'} %}`

Comment (jesseyuen) [reply app/views/_templates/_new-record.html]: You can? That's a bit fancy!

Comment (jesseyuen) [reply app/views/_templates/_new-record.html]: IT WORKS!

**Review by edwardhorsford — APPROVED** [review]

**Pull request merged**